src/user/hooks/useLoginForm.ts

```diff
@@ -26,9 +26,16 @@ export const useLoginForm = () => {
     await login({ email: userName, password });
   };
 
+  const handleKeyDown = (event: React.KeyboardEvent) => {
+    if (isSubmitDisabled || error) return;
+    if (event.key !== 'Enter' && event.key !== 'NumpadEnter') return;
+    handleSubmit();
```

Comment:
Please add an empty line here. (it will be one of the eslint rules soon)

Reply:
Line added